.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,28 @@
+<!-- 
+Insert issue number that this PR fixes (if any) just after 'Fixes #'.
+If this PR does not fix an existing issue, consider opening one or remove 'Fixes #' from the PR description.
+-->
 Fixes #
 
 Changes made in this Pull Request:
- - 
+<!-- Describe the changes that this PR makes. If applicable, use the following bullet list. --> 
+-
 
-
-PR Checklist
-------------
- - [ ] Tests?
- - [ ] Docs?
- - [ ] CHANGELOG updated?
+## PR Checklist
+<!-- Please use the following checklist to ensure the PR is ready to be reviewed/merged. -->
  - [ ] Issue raised/referenced?
+ - [ ] Tests updated/added?
+ - [ ] Documentation updated/added?
+ - [ ] `package/CHANGELOG` file updated?
+ - [ ] Is your name in `package/AUTHORS`? (If it is not, add it!)
+
+## Developers Certificate of Origin
+<!-- 
+CHECKING THE FOLLOWING BOX (BY ADDING 'x' INSIDE THE BRACKETS `[x]`) IS MANDATORY FOR ALL PRs.
+-->
+
+In order for this PR to be merged you **must certify that you are able to submit your code to be included in MDAnalysis**. Read the links and then check the box if you agree.
+
+Code in MDAnalysis is under [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE).
 
-## Developers certificate of origin
-- [ ] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).
+- [ ] I certify that I can legally submit this code contribution as described in the [**Developer Certificate of Origin**](https://developercertificate.org/)
```

Comment:
This is very much just a nit pick - "legally" can be a bit jarring, the wording can work without too it if you feel like it.

Reply:
Good point, I'll remove it.